Project 1/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>  
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel= "stylesheet" href= "styles.css">
+    <title>Intelligent Octopus</title>
+</head>
+<body>
+        <body style="background-color:purple;">
+    <div class="card"> <h1>Yes, the Octopus Is Smart as Heck. But Why?</h1>
```

Comment:
You open a `div` here on line 12, but never close it. This was [previously noted](https://github.com/drealtimonster/gd2/pull/1/files?file-filters%5B%5D=.css&file-filters%5B%5D=.html#r258931214).